mri/optimizers/utils/cost.py

```diff
@@ -7,9 +7,131 @@
 # for details.
 ##########################################################################
 
+
 """
-This module contains shortcuts to cost classes.
+Different cost functions for the optimization.
 """
 
-# Package import
-from mri.reconstruct.cost import DualGapCost, GenericCost
+
```

Comment:
This is direct copy, the earlier movewas done to a shortcut file, (which just had imports)